ql.mm

```diff
@@ -10305,84 +9783,82 @@ Kalmbach axioms (soundness proofs) that require WOML
-     to 0, and j is set to 1.  (Contributed by Roy F. Longton, 3-Jul-05.) $)
+     to 0, and j is set to 1.  (Contributed by Roy F. Longton, 1-Jul-2005.)
+     (Revised by Roy F. Longton, 3-Jul-2005.) $)
   lem4.6.6i0j1 $p |- ( ( a ->0 b ) v ( a ->1 b ) ) = ( a ->0 b ) $=
     ( wn wo wa wi0 wi1 leid lear lelor lel2or leo lebi df-i0 df-i1 2or 3tr1 ) A
     CZBDZRABEZDZDZSABFZABGZDUCUBSSSUASHTBRABIJKSUALMUCSUDUAABNZABOPUEQ $.
-    $( [3-Jul-05] $) $( [1-Jul-05] $)
 
   $( Equation 4.14 of [MegPav2000] p. 23.  The variable i in the paper is set
-     to 0, and j is set to 2.  (Contributed by Roy F. Longton, 3-Jul-05.) $)
+     to 0, and j is set to 2.  (Contributed by Roy F. Longton, 1-Jul-2005.)
+     (Revised by Roy F. Longton, 3-Jul-2005.) $)
   lem4.6.6i0j2 $p |- ( ( a ->0 b ) v ( a ->2 b ) ) = ( a ->0 b ) $=
     ( wn wo wa wi0 wi2 leid leor leao1 lel2or leo lebi df-i0 df-i2 2or 3tr1 ) A
     CZBDZBRBCZEZDZDZSABFZABGZDUDUCSSSUBSHBSUABRIRTBJKKSUBLMUDSUEUBABNZABOPUFQ
     $.
-    $( [3-Jul-05] $) $( [1-Jul-05] $)
 
   $( Equation 4.14 of [MegPav2000] p. 23.  The variable i in the paper is set
-     to 0, and j is set to 3.  (Contributed by Roy F. Longton, 3-Jul-05.) $)
+     to 0, and j is set to 3.  (Contributed by Roy F. Longton, 1-Jul-2005.)
+     (Revised by Roy F. Longton, 3-Jul-2005.) $)
   lem4.6.6i0j3 $p |- ( ( a ->0 b ) v ( a ->3 b ) ) = ( a ->0 b ) $=
     ( wn wo wa wi0 wi3 leid leao1 lel2or lear leo lebi df-i0 df-i3 2or 3tr1 ) A
     CZBDZRBEZRBCZEZDZASEZDZDZSABFZABGZDUGUFSSSUESHUCSUDTSUBRBBIRUABIJASKJJSUELM
     UGSUHUEABNZABOPUIQ $.
-    $( [3-Jul-05] $) $( [1-Jul-05] $)
 
   $( Equation 4.14 of [MegPav2000] p. 23.  The variable i in the paper is set
-     to 0, and j is set to 4.  (Contributed by Roy F. Longton, 3-Jul-05.) $)
+     to 0, and j is set to 4.  (Contributed by Roy F. Longton, 1-Jul-2005.)
+     (Revised by Roy F. Longton, 3-Jul-2005.) $)
   lem4.6.6i0j4 $p |- ( ( a ->0 b ) v ( a ->4 b ) ) = ( a ->0 b ) $=
     ( wn wo wi0 wi4 leid leao4 leao1 lel2or lea leo lebi df-i0 df-i4 2or 3tr1
     wa ) ACZBDZABRZSBRZDZTBCZRZDZDZTABEZABFZDUHUGTTTUFTGUCTUEUATUBBASHSBBIJTUDK
     JJTUFLMUHTUIUFABNZABOPUJQ $.
-    $( [3-Jul-05] $) $( [1-Jul-05] $)
 
   $( Equation 4.14 of [MegPav2000] p. 23.  The variable i in the paper is set
-     to 1, and j is set to 0.  (Contributed by Roy F. Longton, 3-Jul-05.) $)
+     to 1, and j is set to 0.  (Contributed by Roy F. Longton, 1-Jul-2005.)
+     (Revised by Roy F. Longton, 3-Jul-2005.) $)
   lem4.6.6i1j0 $p |- ( ( a ->1 b ) v ( a ->0 b ) ) = ( a ->0 b ) $=
     ( wn wa wo wi1 wi0 lear lelor df-le2 df-i1 df-i0 2or 3tr1 ) ACZABDZEZOBEZER
     ABFZABGZETQRPBOABHIJSQTRABKABLZMUAN $.
-    $( [3-Jul-05] $) $( [1-Jul-05] $)
 
   $( Equation 4.14 of [MegPav2000] p. 23.  The variable i in the paper is set
-     to 1, and j is set to 2.  (Contributed by Roy F. Longton, 3-Jul-05.) $)
+     to 1, and j is set to 2.  (Contributed by Roy F. Longton, 1-Jul-2005.)
```

Comment:
The Contributed line should be 3-Jul-2005 and there should be no Revised by. I realize that the original has both dates but there's no point in having two dates two days apart.